homeassistant/components/velux/cover.py

```diff
@@ -131,3 +152,63 @@ async def async_set_cover_tilt_position(self, **kwargs):
         await self.node.set_orientation(
             orientation=orientation, wait_for_completion=False
         )
+
+
+class VeluxWindow(VeluxCover):
+    """Representation of a Velux window."""
+
+    def __init__(self, hass: HomeAssistant, node: Node) -> None:
+        """Initialize Velux window."""
+        super().__init__(node)
+        self._hass = hass
+        self._extra_attr_limitation_min: int | None = None
+        self._extra_attr_limitation_max: int | None = None
+
+        self.coordinator = DataUpdateCoordinator(
+            self._hass,
+            _LOGGER,
+            name=self.unique_id,
+            update_method=self.async_update_limitation,
+            update_interval=DEFAULT_SCAN_INTERVAL,
+        )
+
+    async def async_init(self):
+        """Async initialize."""
+        return await self.coordinator.async_config_entry_first_refresh()
+
+    async def async_update_limitation(self):
+        """Get the updated status of the cover (limitations only)."""
+        async with self._hass.data[DATA_VELUX].semaphore_poll_data:
+            try:
+                limitation = await self.node.get_limitation()
+                self._extra_attr_limitation_min = limitation.min_value
+                self._extra_attr_limitation_max = limitation.max_value
+            except PyVLXException:
+                _LOGGER.error("Error fetch limitation data for cover %s", self.name)
+
+    @property
+    def extra_state_attributes(self) -> dict[str, int | None]:
+        """Return the state attributes."""
+        return {
+            "limitation_min": self._extra_attr_limitation_min,
+            "limitation_max": self._extra_attr_limitation_max,
```

Comment:
How is it useful to export these extra attributes? What do they mean, and how can they be used by our users?

Reply:
> Hey, those values are coming from the rainsensor attached to the velux integra
> 
> limitation_min => 0 , no rain limitation_min => 93 => the velux can only open fot 7 % , so it means, its raining
> 
> Usefull for automations to close screens/curtains

Other home automation solutions like Fhem has the same feature. [DOC](https://wiki.fhem.de/wiki/Velux_KLF200#Auswerten_der_Regensensoren)

Reply:
If this is indeed needed, please add a documentation PR which explains the new attributes here: https://www.home-assistant.io/integrations/velux/

Reply:
Done.
Updated PR's description.

I have another idea to prevent the polling by default. I can add a service call like  
`
velux.update_limitation
   entity_id: cover.xxxx
` 
and then the user can decide for himself with an automation if and how often update the sensor.
What do you think about this idea?

Reply:
Sounds good :-)

Reply:
Thank you for your review and reply.
I added some answers in the documentation PR.